tests/test_models.py

```diff
@@ -29,3 +31,13 @@ def test_daily_mean_integers():
     # Need to use Numpy testing functions to compare arrays
     npt.assert_array_equal(daily_mean(test_input), test_result)
 
+
+@pytest.mark.parametrize('data, expected_standard_deviation', [
+    ([0, 0, 0], 0.0),
+    ([1.0, 1.0, 1.0], 0),
+    ([0.0, 2.0], 1.0)
+])
```

Comment:
Maybe it is a good idea to include a test for negative input numbers? It should not happen as inflammation is not negative, but nevertheless it might be a good idea.

Let me know if you agree

Reply:
Fixed via #40702d1